src/config/2020.json

```diff
@@ -1191,6 +1191,14 @@
       "avatar_url": "https://avatars2.githubusercontent.com/u/24447527?v=4&s=200",
       "github": "raghuramakrishnan71"
     },
+    "raphlinus": {
+      "name": "Raph Levien",
+      "teams": [
+        "authors"
+      ],
+      "avatar_url": "https://avatars2.githubusercontent.com/u/242367?v=4&s=200",
+      "github": "raphlinus"
```

Comment:
@raphlinus feel free to also add your website, Twitter, or LinkedIn as applicable.